app/src/main/res/layout-land/hints_summary.xml

```diff
@@ -10,106 +10,105 @@
       name="isListExpanded"
       type="Boolean" />
 
-     <variable
+    <variable
       name="viewModel"
       type="org.oppia.android.app.hintsandsolution.HintsViewModel" />
   </data>
 
-    <LinearLayout
-      android:id="@+id/hint_summary_container"
+  <LinearLayout
+    android:id="@+id/hint_summary_container"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginTop="32dp"
+    android:orientation="vertical">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
-      android:layout_marginTop="32dp"
-      android:orientation="vertical">
+      android:layout_marginStart="72dp"
+      android:layout_marginEnd="72dp"
+      android:layout_marginBottom="20dp"
+      android:orientation="horizontal">
 
-      <androidx.constraintlayout.widget.ConstraintLayout
-        android:layout_width="match_parent"
+      <TextView
+        android:id="@+id/hint_title"
+        style="@style/Heading2"
+        android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginStart="72dp"
-        android:layout_marginEnd="72dp"
-        android:layout_marginBottom="20dp"
-        android:orientation="horizontal">
+        android:layout_gravity="center"
+        android:layout_marginTop="12dp"
+        android:layout_marginBottom="12dp"
+        android:textColor="@color/oppiaPrimaryText"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
 
-        <TextView
-          android:id="@+id/hint_title"
-          android:layout_width="wrap_content"
-          android:layout_height="wrap_content"
-          android:layout_gravity="center"
-          android:layout_marginTop="12dp"
-          android:layout_marginBottom="12dp"
-          android:fontFamily="sans-serif-medium"
-          android:textColor="@color/oppiaPrimaryText"
-          android:textSize="20sp"
-          app:layout_constraintStart_toStartOf="parent"
-          app:layout_constraintTop_toTopOf="parent" />
+      <Button
+        android:id="@+id/reveal_hint_button"
+        style="@style/StateButtonActive"
+        android:layout_width="wrap_content"
```

Comment:
You can remove all the features that are getting enforced through the style. Things like `layout_width`, `layout_height` and `textColor` can be removed.